MapNotepad/MapNotepad/MapNotepad/ViewModel/BaseViewModel.cs

```diff
@@ -0,0 +1,50 @@
+﻿using MapNotepad.View;
+using Prism.Mvvm;
+using Prism.Navigation;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MapNotepad.ViewModel
+{
+
+    class BaseViewModel : BindableBase
```

Comment:
In BaseViewModel you shouldn't create method for navigation for every page, it is redundant.